config/initializers/verify.rb

```diff
@@ -0,0 +1,3 @@
+Verify.configure do |config|
```

Comment:
👍 

Feels like (once it's all done and dusted) this could be something we could break out into a gem